doc/introduction/data.rst

```diff
@@ -0,0 +1,162 @@
+-----------------------------
+
+We can access data of a desired type with the :func:`~pennylane.data.load` function. This downloads the desired
+dataset or loads it from local storage if previously downloaded.
+
+To specify the dataset to be loaded, the data category (``data_name``) must be
+specified, alongside category-specific keyword arguments. For the full list
+of available datasets, please see the dataset library documentation.
+The :func:`~pennylane.data.load` returns a list with the desired data which can then be used within
+the usual PennyLane workflow.
+
+.. code-block:: python
+
+    >>> H2_dataset = qml.data.load(data_name="qchem", molname="H2", basis="6-31G", bondlength="0.46")
+    >>> print(H2_dataset)
+    [<pennylane.data.dataset.Dataset object at 0x7f14e4369640>]
+
+When we only want to download portions of a large dataset, we can specify the desired properties  (referred to as _attributes_).
+For example, we can download only the molecule and hamiltonian of a dataset as follows:
+
+.. code-block:: python
+
+    >>> H2_hamiltonian = qml.data.load(data_name='qchem',molname='H2', basis='6-31G', bondlength='0.46', attributes=['molecule','hamiltonian'])
+    >>> H2_hamiltonian
+    <Hamiltonian: terms=185, wires=[0, 1, 2, 3, 4, 5, 6, 7]>
+
+To determine what attributes are available for a type of dataset, we can use the function :func:`~pennylane.data.list_attributes`:
+
+.. code-block:: python
+    >>> qml.data.list_attributes(data_name='qchem')
+    ['molecule',
+    'hamiltonian',
+    'wire_map',
+    ...
+    'vqe_params',
+    'vqe_circuit']
+
+Using Datasets in PennyLane
+---------------------------
+
+Once loaded, one can access properties of the datasets (referred to as _attributes_):
+
+.. code-block:: python
+
+    >>> H2_dataset[0].molecule
+    <pennylane.qchem.molecule.Molecule object at 0x7f890b409280>
+    >>> H2_dataset[0].hf_state
+    [1 1 0 0 0 0 0 0]
+
+The loaded data items are fully compatible with PennyLane. We can therefore
+use them directly in PennyLane workflow as follows:
+
+.. code-block:: python
+
+    >>> dev = qml.device('default.qubit',wires=H2_dataset[0].hamiltonian.wires)
+    >>> @qml.qnode(dev)
+    ... def circuit():
+    ...     return qml.expval(H2_dataset[0].hamiltonian)
+    >>> print(circuit())
+    2.173913043478261
+
+Dataset Structure
+-----------------
+
+You can call the 
+:func:`~.pennylane.data.list_datasets` function to get a snapshot of the currently available data.
+This function returns a nested dictionary as we show below. Note that this example limits the results
+of the function calls for clarity and that as more data becomes available, the results of these
+function calls will change.
+
+.. code-block:: python
+
+    >>> available_data = qml.data.list_datasets()
+    >>> available_data.keys()
+    dict_keys(['qspin', 'qchem'])
+    >>> available_data['qchem']
+    dict_keys(['H2', 'HeH'])
+    >>> available_data['qchem']['H2']
+    dict_keys(['6-31G'])
```

Comment:
this isn't actual output right? did you ask for `.keys()` locally and not write it in here?

Reply:
Yup, these lines should have `.keys()` at the end. Will edit.